client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -184,22 +220,42 @@ module.exports = React.createClass( {
 
 		return (
 			<span className="plugin-install-button__install">
-				<a onClick={ this.installAction } className="button is-primary" disabled={ this.props.isInstalling } >
+				<Button onClick={ this.installAction } is-primary={ true } disabled={ isInstalling } >
 					{ label }
-				</a>
+				</Button>
 			</span>
 		);
-	},
+	}
 
-	render: function() {
-		if ( this.props.selectedSite.unreachable ) {
+	render() {
+		const { selectedSite } = this.props;
+		if ( selectedSite.unreachable ) {
 			return this.renderUnreachableNotice();
 		}
-		if ( ! this.props.selectedSite.canUpdateFiles ) {
+		if ( ! selectedSite.canUpdateFiles ) {
 			return this.renderDisabledNotice();
 		}
 
 		return this.renderButton();
 	}
-} );
+}
+
+PluginInstallButton.displayName = 'PluginInstallButton';
+PluginInstallButton.propTypes = {
+	selectedSite: PropTypes.object.isRequired,
+	isEmbed: PropTypes.bool,
+	isInstalling: PropTypes.bool,
+	plugin: PropTypes.object.isRequired,
+	notices: React.PropTypes.object,
+	isMock: PropTypes.bool,
+	disabled: React.PropTypes.bool,
+};
+
+export default connect(
+	null,
+	{
+		recordGoogleEvent,
+		recordTracksEvent
+	}
+)( localize( PluginInstallButton ) );
 
```

Comment:
One new line at the end of the file should be enough. Currently I can see 2 of them.